### PR TITLE
Raison d'etre

### PR DESCRIPTION
**Pull request by easternbloc**
Just a few words to explain why we have created hof-logger

### CODE REVIEW

**Comment by easternbloc** [comment]
@JoeChapman I, @joefitter and @danielepolencic needs admin for this project kthxbye

**Comment by JoeChapman** [comment]
@easternbloc you got it

**Comment by easternbloc** [comment]
@JoeChapman 💌

File: README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # hof-logger
-Common logger pattern
+A Common logger pattern to be used across HOF projects.
+
+Although this modules doesn't appear to do very much the intention is to ensure we have a common logging approach across Home Office Forms.
```
Comment (JoeChapman) [inline README.md:5]: Very brief, can you please elaborate @easternbloc

**Pull request closed without merging**